torch/ao/nn/intrinsic/modules/fused.py

```diff
@@ -144,3 +145,13 @@ def __init__(self, linear, tanh):
             'Incorrect types for input modules{}{}'.format(
                 type(linear), type(tanh))
         super().__init__(linear, tanh)
+
+class ConvAdd2d(_FusedModule):
+    r"""This is a sequential container which calls the Conv2d modules with extra Add.
+    During quantization this will be replaced with the corresponding fused module."""
+    def __init__(self, add, conv):
+        super().__init__(conv)
+        self.add = add
+
+    def forward(self, x1, x2):
+        return self.add(self[0](x1), x2)
```

Comment:
also this feels a bit hacky, i'm wondering how we can extend _FusedModule to support this use case..